include/proxy-wasm/limits.h

```diff
@@ -23,3 +23,9 @@
 #ifndef PROXY_WASM_HOST_MAX_WASM_MEMORY_SIZE_BYTES
 #define PROXY_WASM_HOST_MAX_WASM_MEMORY_SIZE_BYTES (1024 * 1024 * 1024)
 #endif
+
+// Maximum allowed random_get buffer size. This value is consistent with
+// the JavaScript Crypto.getRandomValues() maximum buffer size.
```

Comment:
could you give us a reference to this JavaScript's behavior?

Reply:
Sure, just added a link, thanks!

Reply:
FWIW, I'm not sure how relevant this is in this context, since we're not implementing JavaScript API.